trustyaiexplainability-core/src/test/java/org/kie/trustyai/explainability/local/lime/optim/LimeConfigEntityFactoryTest.java

```diff
@@ -49,24 +50,24 @@ void testConversion() {
     @Test
     void testWeightingEntities() {
         List<? extends LimeConfigEntity> entities = LimeConfigEntityFactory.createWeightingEntities(new LimeConfig());
-        assertThat(entities).isNotNull().hasSize(1);
+        Assertions.assertThat(entities).isNotNull().hasSize(1);
```

Comment:
This change should not be necessary, there is already an import static of `assertThat`